packages/database/src/TupaiaDatabase.js

```diff
@@ -51,6 +52,10 @@ const VALID_COMPARISON_TYPES = ['where', 'whereBetween', 'whereIn', 'orWhere'];
 // keeping all the tests passing
 const HANDLER_DEBOUNCE_DURATION = 250;
 
+// turn off parsing of timestamp (not timestamptz), so that it stays as a sort of "universal time"
+// string, independent of timezones, rather than being converted to local time
+pg.types.setTypeParser(1114, val => val);
```

Comment:
I don't love the magic number here, are these ids enumerated somewhere?

Reply:
Yep, in a more recent version of `node-pg` 😞 

I can check out what's required to upgrade

Reply:
I've looked through the breaking changes from pg 7 -> 8 and we're safe on all counts, so I'm going to upgrade and use the oid constants provided there.